dkg/pedersen/pedersen_test.go

```diff
@@ -470,3 +479,44 @@ type fakeSigner struct {
 func (s fakeSigner) GetPublicKey() crypto.PublicKey {
 	return ed25519.NewPublicKeyFromPoint(s.pubkey)
 }
+
+// decryptReencrypted helps to decrypt a reencrypted message.
+func decryptReencrypted(Cs []kyber.Point, XhatEnc kyber.Point, dkgPk kyber.Point, Sk kyber.Scalar) (msg []byte, err error) {
```

Comment:
Could we spin this out in its own library ?

Reply:
I'll do that in next step.